v3/setup/images-control.sh

```diff
@@ -6,11 +6,11 @@ if [ "${NODE_ROLE}" != "control" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]; then
```

Comment:
Super minor nitpick, but in other places we've used `/bootstrap.service/flight-director` (https://github.com/adobe-platform/mesos-systemd/blob/master/v3/setup/flight-director.sh#L8). Would it make more sense to call this `bootstrap.service/images-control` or even `bootstrap.service/images/control`?